### DISCUSSION TITLE
Shaky FF OSD on A500 plus

### DISCUSSION THREAD

**Discussion by robxbl69** [keirf/FF_OSD#56] [Q&A]
Hi. I've already got FF OSD on my A600 and it's rock steady. I've just added it to my A500 plus and the bottom of the OSD quivers when there's high contrast or a lot of motion. Any thoughts?

Thanks, Rob.

**Comment by keirf** [comment]
Try entering OSD configuration and moving the OSD box a little left or right. Sometimes you can happen upon a slightly more stable position. Unfortunately the box isn't sync'ed tot he host pixel clock, and this can make it a bit juddery on digital displays.

**Comment by robxbl69** [comment]
It's exactly the same on a CRT (1084S).

I was going to try re-routing and maybe shortening the video signal from
the "blue pill" just in case it's picking up noise from somewhere?

**Comment by keirf** [comment]
Yes that's interesting. It could indeed be noise.

**Reply by robxbl69** [reply]
I had the video and sync running down the middle of the board. Just replaced wiring so now running along the back edge of the A500 (along the top edge of the shielding under the motherboard) and the picture is super stable now :) Waving the wires over the top of Paula seemed to create noise in the video signal.
